fs/indexed-pkg-importer/src/index.ts

```diff
@@ -136,11 +136,10 @@ function createCloneFunction (): CloneFunction {
       try {
         reflinkFileSync(fr, to)
       } catch (err: unknown) {
-        assert(util.types.isNativeError(err))
         // If the file already exists, then we just proceed.
         // This will probably only happen if the package's index file contains the same file twice.
         // For instance: { "index.js": "hash", "./index.js": "hash" }
-        if (!err.message.startsWith('File exists') && !err.message.includes('-2147024816')) throw err
+        if (!util.types.isNativeError(err) || !('code' in err) || err.code !== 'ENOENT') throw err
```

Comment:
I'm dumb.

```suggestion
        if (!util.types.isNativeError(err) || !('code' in err) || err.code !== 'EEXIST') throw err
```

Reply:
Could we add the OS error code for now for reflinks? it's an edgecase but as we already had the same issue before

Reply:
It's just a refactor. There's no rush to merge this. The old code still works just fine.